themes/SuiteP/css/suitep-base/editview.scss

```diff
@@ -5448,7 +5429,7 @@ body.yui-skin-sam.masked .yui-panel-container.yui-dialog.yui-simple-dialog.shado
   top: 100px;
   padding: 20px;
   width: 100%;
-  position: fixed;
+  z-index: 103100;
```

Comment:
Is this required? ideally would want to avoid setting more crazy z-indexes if possible.